Tests/AppTests/Util+ext.swift

```diff
@@ -27,6 +27,10 @@ extension XCTestCase {
         ProcessInfo.processInfo.environment.keys.contains("GITHUB_WORKFLOW")
     }
     
+    var runImageSnapshotTests: Bool {
+        ProcessInfo.processInfo.environment.keys.contains("RUN_IMAGE_SNAPSHOT_TESTS")
+    }
```

Comment:
We were previously using `GITHUB_WORKFLOW` as a signal not to run the snapshot image tests. Because we're now also checking `isRunningInCI` for the signing tests, it's not possible to control both the singing and the snapshot tests independently.